lighthouse-cli/performance-experiment/experiment-database/index.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = require('./database');
```

Comment:
any reason to have this file and not just do `const ExperimentDatabase = require('./experiment-database/database');`?